x/payments/handler.go

```diff
@@ -129,7 +141,18 @@ func handleMsgCreatePaymentTemplate(ctx sdk.Context, k Keeper, bk bank.Keeper, m
 	// Submit payment template
 	k.SetPaymentTemplate(ctx, msg.PaymentTemplate)
 
-	return sdk.Result{}
+	ctx.EventManager().EmitEvents(sdk.Events{
+		sdk.NewEvent(
+			types.EventTypeCreatePaymentTemplate,
+			sdk.NewAttribute(types.AttributeKeyCreatorDid, msg.CreatorDid),
```

Comment:
Here we want to output all attributes of the payment template, not just the creator DID.

Reply:
Ok . Added payment template